ClassicPress-research/classic-commerce
# Change WC to CC on shipping zone page

### All Submissions:


### Changes proposed in this Pull Request:

Change one occurrence of WC to CC on the shipping zone page, in the file: html-admin-page-shipping-zones.php

### How to test the changes in this Pull Request:

1. Check out PR with change to html-admin-page-shipping-zones.php file
2. Copy this file to a test site and visit the shipping zone section of settings.

### Other information:


### Changelog entry

Change WC to CC on shipping zone page.
